src/html/amp-validate/amp-validate.html

```diff
@@ -0,0 +1,47 @@
+<link href="../../../bower_components/polymer/polymer.html">
+
+<script src="../../js/AMPSetup/AMPSetup.js"></script>
+<script src="../../js/validation/isHex/validateIsHex.js"></script>
+
+<!--
+    #<amp-validate> 
+
+    Does basic validation on a hexadecimal input to verify that it's the correct format for parsing.
+
+    @demo validate-demo.html
+    -->
+<dom-module id="amp-validate">
+    <script>
+    Polymer({
+        is: "amp-validate",
+        properties: {
+            /** The user's input to be validated */
+            rawInput: {
+                type: String,
+                observer: "_determineIsValid"
+            },
+            /** Whether or not the user's input is valid. Updated after the user hasn't made a keypress in 250ms */
+            invalid: {
+                type: Boolean,
+                value: false,
+                notify: true
+            }
+        },
+
+        _determineIsValid: function(newValue)
+        {
+            this.debounce("input changes", function()
+            {
+                try
+                {
+                    this.invalid = !AMPParse.validateIsHex(this.rawInput);
+                }
+                catch (err)
+                {
+                    this.invalid = true;
+                }
+            }, 250);
+        }
+    })
+    </script>
+</dom-module>
```

Comment:
I'm fine with pushing this if it works, but looking at the examples from polymer, this style seems a little clearer to me:
https://github.com/PolymerElements/paper-input/blob/master/demo/ssn-validator.html
https://github.com/PolymerElements/paper-input/blob/master/demo/ssn-input.html

See demo here:
https://elements.polymer-project.org/elements/paper-input?view=demo:demo/index.html&active=paper-input